rakelib/license.rb

```diff
@@ -0,0 +1,40 @@
+# we need to call exit explicity  in order to set the proper exit code, otherwise
+# most common CI systems can not know whats up with this tests.
+require_relative "default_plugins"
+require "rubygems/specification"
+require "bootstrap/environment"
+
+
+def all_installed_gems
+  ENV["GEM_HOME"] = ENV["GEM_PATH"] = LogStash::Environment.logstash_gem_home
+  Gem.use_paths(LogStash::Environment.logstash_gem_home)
+
+  Gem::Specification.all = nil
```

Comment:
.all is used to get all of the dependencies that is accessible in the current rubygem environments. 

I will look up on enumerable to see what it does exactly